pkg/minikube/command/command_runner.go

```diff
@@ -77,6 +77,8 @@ type Runner interface {
 
 	// Remove is a convenience method that runs a command to remove a file
 	Remove(assets.CopyableFile) error
+
+	ReadableFile(sourcePath string) (assets.ReadableFile, error)
```

Comment:
comment on this interface, what is this method exactly and is this the best name for it ?

Reply:
I split `assets.CopyableFile` to source(`assets.ReadableFile` interface) and target part in `pkg/minikube/assets/vm_assets.go`.
And this function defined to get source file info/io.Readable interface from remote node. hmm any other suggestion?

Reply:
thats fine but add a comment, all public funcs needs to have comment